src/test/kotlin/com/pubnub/contract/channelmetadata/step/WhenSteps.kt

```diff
@@ -0,0 +1,70 @@
+package com.pubnub.contract.channelmetadata.step
+
+import com.pubnub.contract.channelmetadata.state.ChannelMetadataState
+import com.pubnub.contract.state.World
+import io.cucumber.java.en.When
+
+class WhenSteps(
+    private val world: World,
+    private val channelMetadataState: ChannelMetadataState
+) {
+
+    @When("I get the channel metadata")
+    fun i_get_the_channel_metadata() {
+        world.pubnub.getChannelMetadata(channel = channelMetadataState.channelId!!).sync()?.let {
+            channelMetadataState.channelMetadata = it.data
+            world.responseStatus = it.status
+        }
+    }
+
+    @When("I set the channel metadata")
+    fun i_set_the_channel_metadata() {
+        val channelMetadata = channelMetadataState.channelMetadata!!
+        world.pubnub.setChannelMetadata(
+            channel = channelMetadata.id,
+            name = channelMetadata.name,
+            description = channelMetadata.description,
+            custom = channelMetadata.custom,
```

Comment:
Why there is no includeCustom(true) here?

Reply:
I guess it's because nothing said that it should be 🤷

Reply:
I see.